app/models/facility_district.rb

```diff
@@ -28,6 +28,14 @@ def reportable_children
     children
   end
 
+  def facility_regions
+    # this should really be Region.facility_regions.where(source_id: facility_ids)
+    # change while fixing https://app.clubhouse.io/simpledotorg/story/3380/facilitydistrict-should-be-initialized-with-name-not-slug
```

Comment:
I don't want to fix the bug as part of this because this needs to get out soon and I don't want to risk breaking anything else.